File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidPictureError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class InvalidPictureError extends UserFacingError {

    public InvalidPictureError() {
        super(UserFacingErrorMessages.INSTANCE.invalidPicture());
    }

    public InvalidPictureError(Throwable cause) {
        super(UserFacingErrorMessages.INSTANCE.invalidPicture(), cause);
    }
}

File: apollo/src/main/java/io/muun/apollo/domain/errors/InvalidPictureError.kt
package io.muun.apollo.domain.errors


import io.muun.apollo.data.external.UserFacingErrorMessages

class InvalidPictureError: UserFacingError {

    constructor():
        super(UserFacingErrorMessages.INSTANCE.invalidPicture())

    constructor(cause: Throwable):
        super(UserFacingErrorMessages.INSTANCE.invalidPicture(), cause)
}
